Jenkinsfile

```diff
@@ -85,7 +85,7 @@ pipeline {
             }
             container('rancher') {
                 sh "rancher apps delete ${NAME} || true" 
-                sh "sleep 5s" // wait for deletion
+                sh "sleep 15s" // wait for deletion
```

Comment:
why tripple the amount?

Reply:
the deploy failed with a 409 error in one of the builds, same happened in frontend repo where this fixed it